### PR TITLE
images: Improvements to rhel4edge

### PR DESCRIPTION
**Pull request by mvollmer**
Running jobs while filling the disk until overflow at the same time is not a good idea.  In particular, this would often prevent the kdump initrd from being generated, which would then be done instead on every boot of the new image.

 * [x] image-refresh rhel4edge

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh rhel4edge done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel4edge-20230216-122627

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/rhel4edge-20230216-122626.log

**Review by martinpitt — DISMISSED** [review]
Thanks for debugging this!

**Comment by martinpitt** [comment]
Oh noes, this [still does not fix testInsightsStatus](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4418-20230216-125919-cef7d48f-rhel4edge-cockpit-project-cockpit/log.html#269-2) :sob:

**Comment by mvollmer** [comment]
> Grr, now kdump says:
> 
> ```
> Detected change(s) in the following file(s):  /etc/kdump.conf /etc/kdump/post.d/ /etc/kdump/pre.d/
> ```

This happens because of something that ostree.install does, so let's fix that over in Cockpit.

**Comment by cockpituous** [comment]
image-refresh rhel4edge done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel4edge-20230220-122549

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/rhel4edge-20230220-122525.log

**Review by martinpitt — APPROVED** [review]
This LGTM, and tests are green. Maybe that's already sufficient, otherwise let's move https://github.com/cockpit-project/cockpit/pull/18373 to a bots followup PR. Danke!

**Comment by martinpitt** [comment]
Ah no, no tests were triggered :see_no_evil: Doing now.

**Comment by cockpituous** [comment]
image-refresh rhel4edge done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel4edge-20230221-085644

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/rhel4edge-20230221-085656.log

File: images/scripts/ostree.setup [hunk 1]
```diff
@@ -74,6 +74,14 @@ while [ -n "$(systemctl list-jobs --legend=no)" ]; do
     sleep 10
 done
 
+# Installing RPMs in a OSTree image sometimes triggers something that
+# will change the mtime of all of /etc on the next boot.  This in turn
+# will trigger a regeneration of the kdump initrd.  This would happen
+# in each test run, which is wasteful and also might interfere with
+# the test itself.  Let's just switch the automatic regeneration off.
+
+! test -e /etc/kdump.conf || echo "force_no_rebuild 1" >>/etc/kdump.conf
```
Comment (martinpitt) [inline images/scripts/ostree.setup:83]: This would be a little cleaner without the `test` and inside the `rhel4edge` special case, but not important.

**Review by martinpitt — APPROVED** [review]
Thanks! LGTM once green.

**Pull request merged**